assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -253,288 +253,277 @@ db.schema.flyway.scripts.prefix=
+# Defines the way to expose devfile endpoints as end-user's applications, in single-host server strategy.
 # They can either follow the single-host strategy and be exposed on subpaths, or they can be exposed on subdomains.
-# `multi-host`:: expose on subdomains
-# `single-host`:: expose on subpaths
+# Supported exposures:
+# `multi-host`:: Exposes devfile endpoints on subdomains.
+# `single-host`:: Exposes devfile endpoints on subpaths.
 che.infra.kubernetes.singlehost.workspace.devfile_endpoint_exposure=multi-host
 
-# Defines labels which will be set to ConfigMaps configuring single-host gateway.
+# Defines labels which are set to ConfigMaps configuring single-host gateway.
 che.infra.kubernetes.singlehost.gateway.configmap_labels=app=che,component=che-gateway-config
 
-# Used to generate domain for a server in a workspace in case property `che.infra.kubernetes.server_strategy` is set to `multi-host`
+# Generates a domain for a server in a workspace in case the property `che.infra.kubernetes.server_strategy` is set to `multi-host`
 che.infra.kubernetes.ingress.domain=
 
 # Indicates whether {prod-short} server is allowed to create {orch-namespace} for user
 # workspaces, or they're intended to be created manually by cluster administrator.
-# This property is also used by the OpenShift infra.
+# This property is also used by OpenShift infrastructure.
 che.infra.kubernetes.namespace.creation_allowed=true
 
 # Defines Kubernetes default namespace in which user's workspaces are created
 # if user does not override it.
-# It's possible to use `<username>` and `<userid>` placeholders (for example: `che-workspace-<username>`).
-# In that case, new namespace will be created for each user.
-# Used by OpenShift infra as well to specify a Project.
+# It's possible to use `<username>` and `<userid>` placeholders, for example: `che-workspace-<username>`.
+# In that case, new namespace is created for each user.
+# Used by OpenShift infrastructure to specify a Project.
 # The `<username>` or `<userid>` placeholder is mandatory.
 che.infra.kubernetes.namespace.default=<username>-che
 
-# Defines whether che-server should try to label the workspace namespaces.
-# NOTE: It is strongly recommended to keep the value of this property set to true. If false, the new workspace namespaces will not be labeled
-# automatically and therefore not recognized by the Che operator making some features of DevWorkspaces not working.
-# If false, an administrator is required to label the namespaces manually using the labels specified in che.infra.kubernetes.namespace.labels.
-# If you want to manage the namespaces yourself, make sure to follow
+# Defines whether {prod-short} server should try to label the workspace namespaces.
+# NOTE: It's strongly recommended you keep the value of this property set to `true`. If set to `false`, the new workspace namespaces isn't labeled
```

Comment:
Again, this is a conditional sentence with "if..." so you can't use a present tense in the latter half. You want "if ... will not be" as in the original.